web/app/themes/ppj/src/vue/JobSummary.vue

```diff
@@ -47,9 +48,12 @@
     },
 
     methods: {
-      handleViewJobClick: function() {
-        ga('send', 'event', 'ViewJob', 'ViewJob',this.prisonName ,5);
-        window.location = this.url;
+      pushEventToGtm: function() {
+        const gtmEvent = {
+          event: 'view_job_click',
+          prisonName: this.prisonName
+        };
+        window.dataLayer.push(gtmEvent);
       }
```

Comment:
Minor thought - would it be preferable to have a non-generic name for pushEventToGtm given that the functionality of the method is non-generic?

Reply:
I did consider that – and then thought that I might be overthinking it.

Any suggestions on a better name for that method? I was thinking along the lines of `gtmViewJobClickEvent` or something.

Reply:
I was thinking either to have a handleClickEvent wrapper or perhaps call it something like pushJobClickEventToGtm?

Reply:
Or what you suggested :)